sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/implementation/StorageImplUtils.java

```diff
@@ -3,6 +3,8 @@
 
 package com.azure.storage.common.implementation;
 
+import com.azure.core.http.HttpHeaders;
+import com.azure.core.http.HttpRequest;
```

Comment:
Did something pull this dependency in?

Reply:
Just an artefact of me trying something then undoing it